src/main/java/de/srendi/advancedperipherals/common/util/FluidUtil.java

```diff
@@ -0,0 +1,147 @@
+package de.srendi.advancedperipherals.common.util;
+
+import appeng.api.storage.MEStorage;
+import dan200.computercraft.api.lua.LuaException;
+import dan200.computercraft.api.peripheral.IComputerAccess;
+import dan200.computercraft.api.peripheral.IPeripheral;
+import dan200.computercraft.core.apis.TableHelper;
+import de.srendi.advancedperipherals.common.addons.appliedenergistics.AppEngApi;
+import de.srendi.advancedperipherals.common.addons.computercraft.owner.IPeripheralOwner;
+import net.minecraft.ResourceLocationException;
+import net.minecraft.core.Direction;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.IntTag;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.material.Fluid;
+import net.minecraftforge.common.capabilities.ICapabilityProvider;
+import net.minecraftforge.common.util.LazyOptional;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.capability.CapabilityFluidHandler;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+import net.minecraftforge.registries.ForgeRegistries;
+import net.minecraftforge.registries.ForgeRegistryEntry;
+import net.minecraftforge.registries.IForgeRegistry;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class FluidUtil {
+    private static final List<Pair<Predicate<Object>, Function<Object, IFluidHandler>>> EXTRACTORS = new ArrayList<>();
```

Comment:
For what have you added this? Is there a specific reason? If yes, a comment should be here

Reply:
There is no specific reason, I copied from the InventoryUtil class and I forgot to delete this piece. I removed it.

Reply:
Alright